config/scribe.php

```diff
@@ -331,4 +331,23 @@
      *
      */
     'routeMatcher' => \Knuckles\Scribe\Matching\RouteMatcher::class,
+
+    /**
+     * [Advanced usage] Customize how Scribe interacts with your databases.
+     */
+    'databases' => [
+        /**
+         * [Advanced usage] If you would like to enable the use of database drivers which don't support transactions
+         * set this value to true.
+         *
+         * NB: Changes made to your database during generatation WILL persist!
+         */
+        'allow_no_transactions' => false,
```

Comment:
I think `allow_no_transactions` is somewhat confusing. How about `use_without_transactions`?

Also, I'd rather we not nest this. We can also collapse these two configs into one. Something like:

```php
'run_without_database_transactions' => [
    ],
```

Then the user can add whatever connections they're okay with.

Reply:
`use_without_transactions` sounds to me like we are introducing a secondary flow where no transactions will be used period.. perhaps I am nitpicking?! I agree that `allow_no_transactions` is a little confusing but 💯  that `use_without_transactions` is any less. I'll canvas for some opinions today with my team.

In terms of the nesting, this was actually where my thoughts where going at 3 AM this morning. In lue of a better name I 'll work this with the current name.

Reply:
On review and consultation, we like `run_without_database_transactions` when in the array form.